src/components/Table.js

```diff
@@ -6,7 +6,7 @@ import Search from './Search';
 import Columns from './Columns';
 import Rows from './Rows';
 import Pagination from './Pagination';
-import { calculateRows, sortColumn, nextPage, previousPage, goToPage, expandRow, setInputtedPage } from '../actions/TableActions'
+import { calculateRows, sortColumn, nextPage, previousPage, goToPage, expandRow, changeSortFieldAndDirection } from '../actions/TableActions'
```

Comment:
Surprised `setInputtedPage` is removed as it's not in any code changes below.  Is it no longer required?  Is it used anywhere?

Reply:
Is `changeSortFieldAndDirection` your function or existing?
![image](https://user-images.githubusercontent.com/4940106/54624304-ffb23480-4ad1-11e9-821b-4103a5b99bf4.png)

If your function, presumably it should be in this PR?

Reply:
Yeah `setInputtedPage` is an unused import (even in the source master). `changeSortFieldAndDirection` is an existing function that is used to update the column and direction for the sort in the state. The row sorting is performed after that by a separate method. For server side sorting, we need the sort column and direction update  logic to remain the same so I have reused the method.